src/main/resources/hudson/matrix/MatrixChildParametersAction/index.jelly

```diff
@@ -0,0 +1,54 @@
+<!--
+The MIT License
+
+Copyright (c) 2016 CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
+    xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
+    xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
+  <l:layout title="${it.displayName}">
+    <j:invokeStatic var="currentThread" className="java.lang.Thread" method="currentThread" />
+    <j:invoke var="buildClass" on="${currentThread.contextClassLoader}" method="loadClass">
+      <j:arg value="hudson.model.Run"/>
+    </j:invoke>
+    <j:set var="build" value="${request.findAncestorObject(buildClass)}" />
+		<st:include page="sidepanel.jelly" it="${build}" />
+		<l:main-panel>
+			<h1>${%Build} ${build.displayName}</h1>
+			<div class="container-fluid pane-frame">
+				<div class="row">
+					<div class="col-xs-24 pane-header">${%Parameters}</div>
+					<div class="row col-xs-24 pane-content">
+						<table class="pane">
+							<j:forEach var="parameterValue" items="${it.parameters}">
+								<tbody>
+									<st:include it="${parameterValue}"
+									page="value.jelly" />
+								</tbody>
+							</j:forEach>
+						</table>
+					</div>
+				</div>
+			 </div>
+		</l:main-panel>
+  </l:layout>
+</j:jelly>
```

Comment:
This jelly has been copied from core. It's needed here to make it pick the right `it.parameters`. Otherwise the parameters view in child builds page is empty.